### ISSUE TITLE
Manage event data stream from DB

### ISSUE THREAD

**Issue by oubiwann** [hexagram30/mush#42]
We need to create a new server in the mush supervision tree that manages event data.

Blocks #39 

Depends upon: #40, #41 

Tasks:

* [x] Examine simple pubsub options for Erlang apps
   * [x] ~`gen_event`~ not fully applicable
   * [x] `gen_server`
   * [x] @rvirding's [chat_demo](https://github.com/rvirding/chat_demo)  <-- good stuff! This is basically "use `gen_server`"  with lots of the thinking already done 😄 
   * [x] ~https://github.com/uwiger/gproc~ too much
   * [x] ~https://github.com/cabol/erlbus~ too much and maybe not stable enough ...
   * [x] ~https://github.com/cabol/west~ too much
* [ ] Implement a server based upon @rvirding's chat_demo (linked above)
   * [ ] just support running within a single node / release
   * [ ] provide an API for:
      * [ ] processes to subscribe to events
      * [ ] processes to receive events
      * [ ] processes to unsubscribe from events
   * [ ] integrate within existing supervision tree for `hxgm30.mush` or `hxgm30.store`
* [ ] Listen for DB changes
* [ ] Parse the results
* [ ] Perform actions based upon these (update game state? send messages to users? iow, utilize pubsub)
